eng/packages/http-client-csharp-mgmt/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@azure-typespec/http-client-csharp-mgmt",
+  "version": "1.0.0",
+  "type": "module",
+  "main": "dist/emitter/index.js",
+  "exports": {
+    ".": "./dist/emitter/index.js"
+  },
+  "scripts": {
+    "clean": "rimraf ./dist ./emitter/temp && dotnet clean ./generator",
+    "build:emitter": "tsc -p ./emitter/tsconfig.build.json",
+    "build:generator": "dotnet build ./generator",
+    "build": "npm run build:emitter && npm run build:generator",
+    "test:emitter": "vitest run -c ./emitter/vitest.config.ts --passWithNoTests",
+    "test:generator": "dotnet test ./generator --logger \"trx;LogFileName=debug.trx\"",
+    "test": "npm run test:generator && npm run test:emitter",
+    "lint": "eslint . --max-warnings=0",
+    "lint:fix": "eslint . --fix",
+    "prettier:fix": "prettier --write --config .prettierrc.json ./**/*.ts",
+    "prettier": "prettier --check --config .prettierrc.json ./**/*.ts"
+  },
+  "files": [
+    "dist/**"
+  ],
+  "dependencies": {
+    "@typespec/http-client-csharp": "1.0.0-alpha.20250317.2"
```

Comment:
Should this depend on azure emitter instead?

Reply:
changed to depend on both, because we still need the options set methods from `@typespec/http-client-csharp`

Reply:
This should be in the options now, right? Wouldn't we get http-client-csharp as a transitive dependency?

Reply:
updated